### ISSUE TITLE
Consider removing language related to file systems

### ISSUE THREAD

**Issue by nightwatchcyber** [securitytxt/security-txt#201]
Considering the inconsistency between the scope of the file in file systems vs. web servers, perhaps we should consider removing the file systems language from the draft, and defer this to a future draft? This should be similar to #196 

@EdOverflow thoughts?

**Comment by nightwatchcyber** [comment]
As per a private conversation with @EdOverflow, we are going to proceed with this

**Comment by EdOverflow** [comment]
Yeah, just wanted to confirm here so that we have a record of it: @nightwatchcyber and I decided it made sense to address file systems in a future specification.

**Comment by austinsonger** [comment]
I really like their take on it 

https://github.com/disclose/dnssecuritytxt
